www/includes/easyparliament/templates/html/search/results.php

```diff
@@ -0,0 +1,111 @@
+<div class="full-page">
+    <div class="full-page__row search-page <?php if ( !$searchstring ) { ?>search-page--blank<?php } ?>">
+
+        <?php include 'form.php'; ?>
+
+        <?php if ( $searchstring && !isset($warnings) ) { ?>
+        <div class="search-page__section search-page__section--results">
+            <div class="search-page__section__primary">
+                <?php if ( $cons ) { ?>
+                    <?php if ( count($cons) > 1 ) { ?>
+                    <h2>MPs in constituencies matching <em class="current-search-term"><?= $info['s'] ?></em></h2>
```

Comment:
The constituency code will only ever return current constituencies so I don't think this will ever list Former MPs so doesn't seem to be required.